premailer/premailer.py

```diff
@@ -116,6 +116,7 @@ class Premailer(object):
     attribute_name = 'data-premailer'
 
     def __init__(self, html, base_url=None,
+                 preserve_all_links=False,
```

Comment:
I don't like the name. Because the intention of this argument is to **disable** the functionality of messing with links. Perhaps something like `disable_link_rewrites=False`. Does that read better to you?

Reply:
Or `disable_link_rewrites=False`?

Whatever we use, I like booleans to default to False as the good default.

Reply:
Yes, that indeed reads better. I will fix that.